AppController/djinn.rb

```diff
@@ -4819,13 +4819,10 @@ def check_stopped_apps()
           Djinn.log_debug("Calling AppManager to stop app #{app}.")
           app_manager = AppManagerClient.new(my_node.private_ip)
           begin
-            if app_manager.stop_app(app)
-              Djinn.log_info("Asked AppManager to shut down app #{app}.")
-            else
-              Djinn.log_warn("AppManager is unable to stop app #{app}.")
-            end
-          rescue FailedNodeException
-            Djinn.log_warn("Failed to talk to AppManager about stopping #{app}.")
+            app_manager.stop_app(app)
+            Djinn.log_info("Asked AppManager to shut down app #{app}.")
```

Comment:
We don't have a way of knowing whether the termination will succeed since it's done asynchronously.